ui/desktop/app/routes/scopes/scope/projects/targets/index.js

```diff
@@ -147,10 +152,13 @@ export default class ScopesScopeProjectsTargetsIndexRoute extends Route {
 
   resetController(controller, isExiting, transition) {
     const { to } = transition;
-    // Reset the scopes query param when changing org scope
+    // Reset the query params when changing org scope
     if (!isExiting && to.queryParams.scopes === '[]') {
```

Comment:
Why do we only check the scopes param?

Reply:
Sorry, this is a little confusing. I don't think I need to check for the other params in sessions either. Here I am only checking `scopes` because I am looking at the `transition.to.queryParams` and not the `transition.from.queryParams`. When the scope has been changed in the header, it has no queryParam values at the route level and I am using that to reset the values on the controller. Happy to discuss if there is a better way to do this that isn't as confusing.